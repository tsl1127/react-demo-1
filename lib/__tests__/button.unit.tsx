// import renderer from 'react-test-renderer'  这种写法在jest里会报错，没有默认导出；是由于tsconfig.json里的allowSyntheticDefaultImports为true导致的,让其为false，让esModuleInterop为true，这里这种写法就可以用

// import * as renderer from 'react-test-renderer'
// import * as React from 'react'
import renderer from 'react-test-renderer'
import React from 'react'
import Button from '../button'

describe('button测试',()=>{
    it('是个div',()=>{
        const json = renderer.create(<Button/>).toJSON()  //变为json
        expect(json).toMatchSnapshot()
    })
})