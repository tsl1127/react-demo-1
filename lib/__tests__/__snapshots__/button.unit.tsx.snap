// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button测试 是个div 1`] = `
<div>
  我是Button
</div>
`;
